docs/sdks/js-ts/overview.mdx

```diff
@@ -202,36 +232,100 @@ const input = new Utils.ActionInput()
     .put("input_name_2", "input_value_2")
     .put("input_name_3", "input_value_3")
 
-// retrieve database ID to locate action
-const dbid = kwil.getDBID("public_key", "database_name")
+const actionBody = {
+    dbid,
+    action: "your_action_name",
+    inputs: input
+}
+
+// pass action body to execute method
+const res = await kwil.call(actionBody)
+
+/*
+    res.data = {
+        result: [ query results ],
+    }
+*/
+```
+</TabItem>
+<TabItem value="ts">
+
+```typescript
+import { Utils, Types } from '@kwilteam/kwil-js'
 
-// construct message
-const msg = await kwil
-    .actionBuilder()
-    .dbid(dbid)
-    .name("your_action_name")
-    .concat(input)
-    .buildMsg()
+// begin constructing the values for the action
+const input: Types.ActionInput = new Utils.ActionInput()
+    .put("input_name_1", "input_value_1")
+    .put("input_name_2", "input_value_2")
+    .put("input_name_3", "input_value_3")
 
-// call from kwil network
-const res = await kwil.call(msg)
+const actionBody: Types.ActionBody = {
+    dbid,
+    action: "your_action_name",
+    inputs: input
+}
+
+// pass action body to execute method
+const res = await kwil.call(actionBody)
 
 /*
     res.data = {
         result: [ query results ],
     }
 */
 ```
+</TabItem>
+</Tabs>
 
 #### Select Query
 
-``` javascript
-const dbid = kwil.getDBID("public_key", "database_name")
-const res = await kwil.selectQuery(dbid, "SELECT * FROM users")
+```javascript
+const dbid = kwil.getDBID("public_key", "database_name");
+const res = await kwil.selectQuery(dbid, "SELECT * FROM users");
 
 /*
     res.data = [
         ...
     ]
 */
-```
\ No newline at end of file
+```
+
+## Database Info
+
+### List Databases
+
+```javascript
+const res = await kwil.listDatabases("public_key");
+// res.data = ["db1", "db2", "db3"]
+```
+
+### Read a Deployed Database Schema
+
+```javascript
+const dbid = kwil.getDBID("public_key", "database_name");
```

Comment:
I still think dbid should not require users to initialize a Kwil class.

Reply:
It is also available on the `Utils` namespace. Okay to leave both, or should it just be on `Utils`?

It would be a breaking change to take it off either classes, just FYI.

Reply:
Probably leave it on kwil as to not break anything, but we should use utils in examples.